porch/engine/pkg/kpt/pkgupdate.go

```diff
@@ -198,3 +200,14 @@ func PkgUpdate(ctx context.Context, ref string, packageDir string, opts PkgUpdat
 
 	return nil
 }
+
+// diskPath is an expediant way to get a paths.FileSystemPath from a dir absolute path.
+//
+// an alternative is `d, err := dir.Open(path)` where the returned value can be used
+// in more ways like `fsp, err := content.FileSystem(d)` and `reader, err := content.Reader(d)`
+func diskPath(path string) paths.FileSystemPath {
```

Comment:
Why a private method here and not reuse standard one?

Reply:
Well, the standard one `DiskPath` is placed currently in `internals/types/types.go`... In part that's to be next to `UniquePath` and `DisplayPath`, and also the placement was conservative --- being in internals in case it wasn't something you'd want as public kptlib api.

If you think it should be part of the exposed interface as well, that sounds fine to me. Do you think a place like `paths.DiskPath()` to be next to `paths.FileSystemPath` would be appropriate?

Reply:
Yes. I think it makes more sense if it is next to `paths.FileSystemPath`. It need not be placed with UniquePath as the semantics are different. And to avoid confusion, I was asking to consider changing the name of the FileSystemPath struct in the other PR.